### PR TITLE
lnurl-pay comment support

### PR DESCRIPTION
**Pull request by fiatjaf**
# Description

This can be tried on https://chat.blixtwallet.com/.

This pull request is categorized as a:

- [x] New feature
- [ ] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Other

## Checklist
- [ ] I’ve run `npm run tsc` and make sure my code didn’t produce errors 
- [ ] I’ve run `npm run prettier` and formatted my code correctly

## Testing

If you added new functionality or fixed a bug, did you add new unit tests?

- [x] No, I’m a fool
- [ ] Yes
- [ ] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [ ] LND
- [ ] c-lightning-REST
- [ ] spark
- [ ] Eclair
- [ ] LndHub

### Locales
- [x] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the Zeus [Transfix page](https://www.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [x] Contributors will need to run `npm install` after this PR is merged in
- [x] 3rd party dependencies have been modified:
    * verify that `package.json` and `package-lock.json` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

**Comment by fiatjaf** [comment]
This should be good.

It includes a fix on lnurl-pay description_hash handling and moving of the `getPayReqLocal` to the `LndHub` backend as it was only used there.

(But I still think we should use the local bolt11 decoding no matter the backend -- and stop calling the backends for the trivial task of decoding an invoice.)

File: locales/cs.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "models.Payment.forwarded": "Přeposláno",
     "models.Payment.fee": "Poplatek",
     "views.LnurlPay.LnurlPay.amount": "Částka k zaplacení (v satoshi)",
+    "views.LnurlPay.LnurlPay.comment": "Komentář",
```
Comment (kaloudis) [inline locales/cs.json:34]: Oy, we want to make locale changes in Transifex https://www.transifex.com/ZeusLN/zeus/

Comment (fiatjaf) [reply locales/cs.json:34]: I am not able to add new strings there. What is the procedure?

Comment (kaloudis) [reply locales/cs.json:34]: For each language you want to add strings to you need to request editor access. Just ping me once the requests are in and I'll approve it.

Sorry the procedure is a pain, there's not really a better way to do this ATM

Comment (ZeusLN) [reply locales/cs.json:34]: Also note, you cannot add translations until the english string has been added, so modify the PR to only include en.json, then I'll merge and update the translation file, THEN you'll be able to make your additions

Comment (fiatjaf) [reply locales/cs.json]: Ok, I did that.
Now I wait for you to do something and then go on Transifex?

**Pull request merged**